win32/src/file_win32.c

```diff
@@ -60,4 +41,472 @@ static void on_file_io_complete_win32(
     ULONG io_result,
     ULONG_PTR number_of_bytes_transferred,
     PTP_IO io
-);
\ No newline at end of file
+);
+
+static VOID NTAPI on_close_threadpool_group_member(
+     PVOID object_context, /*what was passed @  CreateThreadpoolIo*/
+     PVOID cleanup_context /*what was passed @  CloseThreadpoolCleanupGroupMembers*/
+)
+{
+    (void)object_context;
+    (void)cleanup_context;
+}
+
+IMPLEMENT_MOCKABLE_FUNCTION(
+    ,
+    FILE_HANDLE, file_create,
+    EXECUTION_ENGINE_HANDLE, execution_engine,
+    const char*, full_file_name,
+    FILE_REPORT_FAULT, user_report_fault_callback,
+    void*, user_report_fault_context

```

Comment:
Read and write are all on one line. I think we should either have it all written like this, or all on one line. Either way. Consistency ftw!

Reply:
agreed, changing everything to single line

---
In reply to: [456709693](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456709693) [](ancestors = 456709693)